### PR TITLE
 Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
- [x] Create a table named animals with the following columns:

    - id: integerheavy_check_mark

   -  name: stringheavy_check_mark

   -  date_of_birth: dateheavy_check_mark

    - escape_attempts: integerheavy_check_mark

    - neutered: booleanheavy_check_mark

    - weight_kg: decimalheavy_check_mark

 🟢  Copy the SQL you used in the previous point into the schema.sql file in the generated repository.

- [x] Insert the following data:

    - Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.heavy_check_mark
    - Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.heavy_check_mark
    - Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.heavy_check_mark
    - Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.heavy_check_mark
   
🟢  Copy the SQL you used in the previous point into the data.sql file in the generated repository. heavy_check_mark

- [x] Write queries for the following:

    - Find all animals whose name ends in "mon".heavy_check_mark
    - List the name of all animals born between 2016 and 2019.heavy_check_mark
    - List the name of all animals that are neutered and have less than 3 escape attempts.heavy_check_mark
    - List date of birth of all animals named either "Agumon" or "Pikachu".heavy_check_markheavy_check_mark
    - List name and escape attempts of animals that weigh more than 10.5kgheavy_check_mark
    - Find all animals that are neutered.heavy_check_mark
   -  Find all animals not named Gabumon.heavy_check_mark
   -  Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)heavy_check_mark

🟢  Copy the SQL you used in the previous point into the queries.sql file in the generated repository.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name Like '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT name from animals WHERE neutered = true  AND escape_attempts < 3;
+SELECT date_of_birth from animals WHERE name In ('Agumon', 'Pikachu');
```
Comment (Emmyn5600) [inline queries.sql]: - [x] Let us use OR here instead of IN for instance, name = 'Agumon' OR name = 'Pikachu'

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered = true;
+SELECT * from animals WHERE name != 'Gabumon';
```
Comment (Emmyn5600) [inline queries.sql]: - [x] Let us use this symbol, `<>`

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name Like '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (Emmyn5600) [inline queries.sql]: - [x] If you decided to use this way, I would recommend you to use the `EXTRACT` function after WHERE.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi,

Good job so far! 👏🏼 @Hamzaoutdoors 

## STATUS: CHANGES REQUESTED :recycle:

### Changes requested.

- _Check the comment under the review._
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: queries.sql (hunk 1 above)
Comment (Hamzaoutdoors) [reply queries.sql]: Done thank you for your feedback 👍🏼

File: queries.sql (hunk 2 above)
Comment (Hamzaoutdoors) [reply queries.sql]: Done thank you for your feedback 👍🏼

File: queries.sql (hunk 3 above)
Comment (Hamzaoutdoors) [reply queries.sql]: Done thank you for your feedback 👍🏼

**Review by AshakaE — APPROVED** [review]
### Hi @Hamzaoutdoors  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**